docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -0,0 +1,122 @@
+[[create-snapshot-api]]
+=== Create index snapshot API
+++++
+<titleabbrev>Create index snapshot</titleabbrev>
+++++
+
+Takes a snapshot of one or more indices. A repository can contain multiple snapshots of the same cluster. Snapshots are identified by unique names in the cluster, such as `snapshot_1`.
+
+////
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_backup
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// TESTSETUP
+////
+
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_backup/snapshot_1?wait_for_completion=true
+-----------------------------------
+
+[[create-snapshot-api-request]]
+==== {api-request-title}
+
+`POST /_snapshot/<repository>/<snapshot-name>`
+
+[[create-snapshot-api-desc]]
+==== {api-description-title}
+
+By default, a snapshot of all open and started indices in the cluster is created. You can change this behavior by specifying the list of indices in the body of the snapshot request.
```

Comment:
I would add a higher-level intro sentence. We also need to mention data streams.
```suggestion
You can use the create snapshot API to create a <<snapshot-restore,snapshot>>, a
backup taken from a running {es} cluster.

By default, a snapshot includes all data streams and open indices in the
cluster, as well as the cluster state.  You can change this behavior by
specifying a list of data streams and indices to back up in the body of the
snapshot request.
```